Sources/SwiftDocC/Semantics/Metadata/PageKind.swift

```diff
@@ -0,0 +1,76 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import Markdown
+
+extension Metadata {
+    /// A directive that allows you to set a page's role and title heading.
+    ///
+    /// The `@PageKind` directive allows you to override a page's title heading to something other
+    /// than its default. The available page kinds are listed in the ``Kind-swift.enum``
+    /// enumeration.
```

Comment:
Now that were autogenerating the user-facing documentation based on these doc comments it would be better to make these kind of statements less attached to the API if possible:

```suggestion
    /// than its default. The available page kinds are `sampleCode` and `article`.
```